### PR TITLE
Style single editorial

### PR DESCRIPTION
**Pull request by mikeyrayvon**

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -234,7 +233,9 @@ Site.Editorial = {
           $('.js-product-button').addClass('u-hidden');
         }
 
-        $('.product-details').show();
+        _this.$productDetails.removeClass('u-invisible');
```
Comment (varffs) [inline js/main.js:238]: why are u using visibility not display?

**Comment by varffs** [comment]
screen shot 2016-08-31 at 14 32 43

These cols need to loose their bottom margin for the content height to be correct

**Comment by varffs** [comment]
Not sure if it was in this branch but the header now has a different height and thus that height needs hard set in the stylus for the calc vertical thing to work as intended. When I did it I went thru the breakpoints and measured it in dev tools, maybe we leave this til later if it might change again?

File: single-editorial.php [hunk 2]
```diff
@@ -28,77 +28,88 @@
     if (!empty($slides)) {
       if ($is_recent) {
 ?>
-
+      
       <!-- Product details container -->
-      <div class="col col-s-12 col-m-3 set-content-height">
+      <div class="col col-s-10 offset-s-1 col-m-3 offset-m-0 column justify-between">
```
Comment (varffs) [inline single-editorial.php]: what is the purpose of `column justify-between`? It doesn't seem to be doing anything that I can see on my local

Comment (mikeyrayvon) [reply single-editorial.php]: ah thats from before we moved the pagination buttons. when the pagination was in that column and needed to stick to the bottom. i'll take those classes out

**Comment by mikeyrayvon** [comment]
lets wait on setting the header heights until later. the margins will need to be adjusted more probably and clay just handed me a redesign of the header to include the shopping cart icon

**Comment by varffs** [comment]
The bottom margin issue too @mikeyrayvon

**Comment by mikeyrayvon** [comment]
what bottom margin issue

**Comment by varffs** [comment]
The issue with the screenshot above: 

`These cols need to loose their bottom margin for the content height to be correct`

**Comment by mikeyrayvon** [comment]
ok then i suppose we take the bottom margin off of all cols and assign it per element by class

**Pull request merged**